stable/mediawiki/templates/deployment.yaml

```diff
@@ -20,13 +20,26 @@ spec:
         app: {{ template "mediawiki.name" . }}
         chart: {{ template "mediawiki.chart" . }}
         release: {{ .Release.Name | quote }}
+{{- if or .Values.podAnnotations .Values.metrics.enabled }}
```

Comment:
`podAnnotation` is not defined in `values.yaml` (and `values-production.yaml` if it exists).

Can you add it and also include the new variable in the README?